tests/spec_tests/query_tests/order_by.yml

```diff
@@ -170,7 +170,7 @@ tests:
 
   - description: missing sorts before null
     current_db: mydb
-    query: "SELECT * FROM nullAndMissing AS nullAndMissing ORDER BY a ASC"
+    query: "SELECT * FROM nullAndMissing AS nullAndMissing ORDER BY a ASC, _id ASC"
     ordered: true
     result:
       - { 'nullAndMissing': { '_id': 0, 'a': null } }
```

Comment:
The description for this test is `missing sorts before null`; however, that's not what the result is.

Reply:
This test is now failing with the following error:

```
[2025/03/07 19:34:05.826] thread 'generated::spec_tests_query_tests_order_by::missing_sorts_before_null' panicked at e2e-tests/src/generated/spec_tests_query_tests_order_by.rs:619:49:
[2025/03/07 19:34:05.826] called `Result::unwrap()` on an `Err` value: Translation(Algebrize(FieldNotFound("_id", Some(["a"]), OrderBy, 0)))
```